src/main/java/de/retest/web/DefaultValuesProvider.java

```diff
@@ -83,9 +83,6 @@ public boolean isDefault( final String tag, final String attribute, final String
 		if ( attributeValue.equals( "auto" ) ) {
 			return true;
 		}
-		if ( attributeValue.equals( "none" ) ) {
```

Comment:
https://www.w3schools.com/cssref/css_default_values.asp there are `none` or is it a placeholder?

Reply:
Hab ich ja eingetragen...

Reply:
in der defaults.yaml.